docs/topics/auth/username.txt

```diff
@@ -117,32 +122,18 @@ Any items in the ``OPTIONS`` in :setting:`AUTH_USERNAME_VALIDATORS` for your
 validator will be passed to the constructor. All constructor arguments should
 have a default value.
 
-Here's a basic example of a validator, with one optional setting::
+Here's a basic example of a validator. This validator extends 
+``django.core.validators.MinLengthValidator``::
 
-    from django.core.exceptions import ValidationError
+    from django.core import validators
     from django.utils.translation import ngettext 
 
-    class UsernameMinimumLengthValidator:
-        def __init__(self, min_length=1):
-            self.min_length = min_length
-
-        def __call__(self, username):
-            if len(username) < self.min_length:
-                raise ValidationError(
-                    ngettext(
-                        'This username is too short. It must contain at least %(min_length)d character.',
-                        'This username is too short. It must contain at least %(min_length)d characters.',
-                        self.min_length,
-                    ),
-                    code='invalid',
-                    params={'min_length': self.min_length},
-                )
-        
+    class UsernameMinimumLengthValidator(validators.MinLengthValidator):
+                
         def help_text(self):
             return ngettext(
-                'Your username must contain at least %(min_length)d 
-                character.',
-                'Your username must contain at least %(min_length)d
-                characters.',
-                self.min_length
-            ) % {'min_length': self.min_length}
+                'Your username must contain at least %(limit_value)d
+                character.','Your username must contain at least 
```

Comment:
I think the other reviewer's comment about the line breaks meant you should get each string on one line, in full. This code as it is wouldn't compile. You don't have to wrap at 79 chars if you're quoting a code sample.

Reply:
fixed.